releases/release-1.31/release-notes/maps/pr-124588-map.yaml

```diff
@@ -0,0 +1,23 @@
+pr: 124588
+releasenote:
+  text: |-
+    ```
```

Comment:
Let's delete this. I think the issue is they didn't include `NONE` in the release note section: https://github.com/kubernetes/kubernetes/pull/124588